libs/controllers/network/__init__.py

```diff
@@ -50,7 +52,7 @@ class INetworkController:
 
     task: asyncio.Task
     callbacks: dict[int, list]
-    queue: list
+    queue: list = []
```

Comment:
does this need to be static as it will be overwritten in \_\_init\_\_ (line 60)

Reply:
The short version of the story is micropython and me.

So here was the scenario. Put the new code on the board > reset board (more often than not by physically re-plugging the cable) > run code > code gives error, but only about 40% of the time.
I didn't write down the exact error, but it basically came down to that queue didn't quite already exist yet on the second thread. Now does this make sense? Not in the slightest, but given the queue a value here seemed to solve the issue.

As to why it is now being double defined, I didn't look at it again after adding this and i kinda just forgot about it.